docs/genomic_file/README.md

```diff
@@ -0,0 +1,36 @@
+# Model Updates
+ 
+## CavaticaTask Generalization
+Up until this point we have only ingested harmonized files from Cavatica. Now we will also receive
+harmonized files from the Broad Institute. 
+
+### Implementation
+We need to rename the models and fields to generalize the CavaticaTask and CavaticaTaskGenomicFile models.
+
+![Model Updates](cav-task-model-update.png)
+
+### Migration
+Migration should be fairly simple. It will consist of alter columns and alter table commands.
+Also, we don't have any data in cavatica_task, cavatica_task_genomic_files tables.
+
+## Multi-specimen Genomic Files
+We decided that we will not have a separate model to represent multi-specimen genomic files. 
+We will keep the many-to-many relationship between biospecimens and genomic files.
+
+## Sequencing Experiments and Harmonized Files
+
+People need to be able to search for files by various sequencing experiment attributes. There are 2 options for enabling this:
+
+Option 1 - Move the relevant fields (i.e. experimental strategy, etc.) on the sequencing experiment to the genomic files.
```

Comment:
@allisonheath will check to see if this is possible

Reply:
Ok, this is a bit trickier than I thought.

For the purposes of this change, all files I think we should carry over:
`experiment_strategy`
`platform`
`instrument_model`
`is_paired_end`

At the moment it's unlikely for these to have different values, but as with many things bioinformatics nothing is impossible. I'd initially say we should have a standard enum to indicate that there's multiple values and document how people could get the specific values based on the BS id. Probably more ideally would be the ability to store the values as arrays, but not sure how feasible that is? 

I also think we should likely move these off of SE entirely and they will exist only on the appropriate GFs and/or RGs:
`max_insert_size`
`mean_insert_size`
`mean_depth`
`total_reads`
`mean_read_length`

cc @yuankunzhu